src/DefaultFileSystem.cpp

```diff
@@ -75,11 +76,35 @@ DefaultFileSystem::Read(const std::string& path) const
   return result;
 }
 
+std::string tmpfile(std::string pathnormalized)
```

Comment:
- it should be in an anonymous namespace
- the argument should be `const std::string&`